test/algorithms/area/area.cpp

```diff
@@ -175,6 +177,34 @@ void test_large_integers()
     BOOST_CHECK_CLOSE(int_area, double_area, 0.0001);
 }
 
+struct precise_cartesian : bg::strategies::detail::cartesian_base
+{
+    template <typename Geometry>
+    static auto area(Geometry const&)
+    {
+        return bg::strategy::area::precise_cartesian<>();
+    }
+};
+
```

Comment:
Maybe you can add one `simplex` case, just the area of a triangle or square, such that we can also judge the basic functionality.
But besides that I'm OK, thanks.

Reply:
good idea, done